### DISCUSSION TITLE
AI in BetterSEQTA+?

### DISCUSSION THREAD

**Discussion by ar-cyber** [SethBurkart123/EvenBetterSEQTA#156] [Ideas]
This would be cool - it could be like an "Ask AI" button in your Window. It'd probably use Gemini because it's free.

It could also be in direct messages.

**Comment by SethBurkart123** [comment]
I do like the idea of having AI in it. Although I feel that some people (especially schools) won't particularly like AI being in it. You can probably use other extensions for using AI. Now a reasonable use-case is using AI to power certain kinds of backend features. Just it can't be things like chat windows or potentially even generated summaries? I mean what are your thoughts on that?

**Comment by ar-cyber** [comment]
I like the AI powering backend features.

**Comment by MEGA-Dawg68** [comment]
I found it even funnier when Education Horizons released an AI to plan lessons for the teachers. 
https://educationhorizons.com/blog/revolutionising-lesson-planning-with-seqta-ai-assist/

I think we should be allowed to use AI

**Comment by ar-cyber** [comment]
> I found it even funnier when Education Horizons released an AI to plan lessons for the teachers. https://educationhorizons.com/blog/revolutionising-lesson-planning-with-seqta-ai-assist/
> 
> I think we should be allowed to use AI

That makes it even more reasonable to add AI to BetterSEQTA+.

**Comment by ar-cyber** [comment]
I think I'll close this as not planned as it's stale.

**Comment by MEGA-Dawg68** [comment]
> I think I'll close this as not planned as it's stale.

fair enough, still a valid request. You could move it to discussions

**Reply by ar-cyber** [reply]
and thats where it is
